components/merkledb/src/access/extensions.rs

```diff
@@ -232,6 +233,16 @@ pub trait AccessExt: Access {
     {
         self.get_or_create_view(addr.into(), index_type).map(drop)
     }
+
+    /// Marks an index with the specified address as removed during migration.
+    fn create_tombstone<I>(self, addr: I)
```

Comment:
I think `create_tombstone` is more explicit (since there is `IndexType::Tombstone`, although it's hidden from the docs). Additionally, "tombstone" term is frequently used in NoSQL DBs, including [RocksDB](https://github.com/facebook/rocksdb/search?q=tombstone&unscoped_q=tombstone).

Reply:
Agree.